scripts/utils/analyze.py

```diff
@@ -79,7 +79,7 @@ def __init__(self, tools, full=False):
                 ],
                 "tools": [],
                 "conflict": None,
-                "ignore": set(['r', 'privfile', 'u', 'pubfile', 's', 'secret', 'g', 'halg', 'n', 'name'])
+                "ignore": set(['r', 'privfile', 'u', 'pubfile', 's', 'secret', 'g', 'halg', 'n', 'name', 'f', 'format'])
```

Comment:
As `tpm2_readpublic` is the only tool in the `object` group that uses `-f`/`--format`, the option analyser complains. Ignoring this should be fine, as the short option name conflict with `tpm2_activatecredential` is resolved and the name is consistent with [`tpm2_verifysignature`](https://github.com/tpm2-software/tpm2-tools/blob/master/man/tpm2_verifysignature.1.md).